specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerActiveConfiguration.json

```
@@ -0,0 +1,171 @@
+      "parameters": [
+        {
+          "$ref": "./network.json#/parameters/ApiVersionParameter"
+        },
+        {
+          "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ResourceGroupNameParameter"
+        },
+        {
+          "$ref": "#/parameters/NetworkManagerNameParameter"
+        },
+        {
+          "$ref": "#/parameters/ListPageSizeParameter"
+        },
+        {
+          "$ref": "#/parameters/ListSkipTokenParameter"
+        },
+        {
+          "name": "region",
+          "in": "query",
+          "description": "Location name",
+          "required": false,
+          "type": "string",
+          "x-ms-parameter-location": "method"
+        }
+      ],
+      "get": {
+        "tags": [
+          "NetworkManagerActiveConfigurations"
+        ],
+        "operationId": "ActiveConfigurations_List",
+        "description": "Lists active configurations in a network manager",
+        "responses": {
+          "200": {
+            "description": "Get successful. The operation returns the specified Managed Network active configuration resource.",
+            "schema": {
+              "$ref": "#/definitions/ActiveConfigurationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Active Configurations": {
+            "$ref": "./examples/NetworkManagerActiveConfigurationList.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ActiveConfiguration": {
+      "properties": {
+        "commitTime": {
+          "type": "string",
+          "description": "Deployment time string",
+          "format": "date-time"
+        },
+        "region": {
+          "type": "string",
+          "description": "deployment region."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./networkManagerEffectiveConfiguration.json#/definitions/EffectiveConfiguration"
+        }
+      ],
+      "description": "Active Configuration."
+    },
+    "ActiveConfigurationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ActiveConfiguration"
+          },
+          "description": "Gets a page of ActiveConfigurations"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets the URL to get the next set of results."
+        }
+      },
+      "description": "Result of the request to list activeConfigurations. It contains a list of groups and a URL link to get the next set of results."
```

Also, should we say it contains a list of active configurations or "groups"?

```suggestion
      "description": "Result of the request to list active configurations. It contains a list of active configurations and a URL link to get the next set of results."
```